src/main/java/com/es/phoneshop/service/ProductService.java

```diff
@@ -0,0 +1,27 @@
+package com.es.phoneshop.service;
+
+import com.es.phoneshop.model.product.CustomProductDao;
+import com.es.phoneshop.model.product.Product;
+import com.es.phoneshop.model.product.ProductDao;
+
+import java.util.List;
+
+public class ProductService {
```

Comment:
service should also follow the "interface driven design"

And a note for future: we will need all the methods that are defined in dao, so please add them as well to the service, and don't forget about throwing the exceptions